src/lighthouse.js

```diff
@@ -47,6 +47,7 @@ const runLighthouse = async (browserPath, url) => {
     });
     const results = await lighthouse(url, {
       port: chrome.port,
+      output: 'html',
```

Comment:
`results.report` is JSON by default. This changes it to HTML.
It doesn't impact the results object which is under `results.lhr`